spec/ivapi/client/account_spec.rb

```diff
@@ -24,6 +24,18 @@
     end
   end
 
+  describe 'account credits' do
+    before(:each) do
+      stub_command('account_credits', count: 10)
+        .to_return(json_response('account_credits.json'))
+      @account_orders = Ivapi.account.credits
+    end
+
+    it 'should return correct credits count' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.